internal/pgadmin/config.go

```diff
@@ -121,8 +150,25 @@ with open('` + settingsAbsolutePath + `') as _f:
     if type(_data) is dict:
         globals().update({k: v for k, v in _data.items() if _conf.fullmatch(k)})
 `
+	// Set pgAdmin's LDAP_BIND_PASSWORD setting if the value was provided via Secret.
+	// As this assignment happens after any values provided via the 'Settings'
+	// ConfigMap loaded above, this value will overwrite any previous configuration
+	// of LDAP_BIND_PASSWORD (that is, last write wins).
+	var addPassword string
+	if ldapBindPassword != nil {
+
+		addPassword = `import os
+if os.path.isfile('` + ldapPasswordAbsolutePath + `'):
```

Comment:
👍🏻 `os.path.isfile` here makes sense in case the secret ref has `optional: true`.

❓ Is it easier to always have this Python code? Can we drop the `if ldapBindPassword != nil`, the append, etc?

Reply:
Yes, testing this out everything worked as expected, code is updated.